cluster.go

```diff
@@ -275,10 +275,16 @@ func (c *Cluster) alertsHandler() {
 			// only the leader handles alerts
 			leader, err := c.consensus.Leader()
 			if err == nil && leader == c.id {
-				logger.Warningf("Peer %s received alert for %s in %s", c.id, alrt.MetricName, alrt.Peer.Pretty())
+				logger.Warningf(
+					"Peer %s received alert for %s in %s",
+					c.id, alrt.MetricName, alrt.Peer)
```

Comment:
nitpick: can we bump the paren to the next line please

Reply:
why not `alrt.Peer.Pretty`?

Reply:
@kishansagathiya `Pretty()` is a method on a `peer.ID`, at this point `alrt.Peer` is now a string

Reply:
right, my bad